packages/server/test/unit/browsers/browsers_spec.js

```diff
@@ -397,6 +397,12 @@ describe('lib/browsers/index', () => {
       })
     })
   })
+
+  context('connectProtocolToBrowser', () => {
+    it('calls connectProtocolToBrowser on the launcher', () => {
+      sinon.s
```

Comment:
is there something missing here?

Reply:
Removed: [73fef85](https://github.com/cypress-io/cypress/pull/27659/commits/73fef8513d89e7d92f15a74d51467f8ad42e086c)